doc/Tex/RAM_SCB_viz.tex

```diff
@@ -1,12 +1,5 @@
-RAM-SCB output can be visualized in a number of different ways depending on
-the user's tastes and preferences.  However, a standard library for opening,
-manipulating, and plotting exists as a sub-module of Spacepy.  Users may
-find this freely available Python library at
-\href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively,
-development versions can be obtained via the project's
-\href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The
-RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
+RAM-SCB output can be visualized in a number of different ways depending on the user's tastes and preferences.  However, a standard library for opening, manipulating, and plotting exists as a sub-module of Spacepy.  Users may find this freely available Python library at \href{http://spacepy.lanl.gov/}{spacepy.lanl.gov}.  Alternatively, development versions can be obtained via the project's \href{http://sourceforge.net/projects/spacepy/}{Source Forge} page.  The RAM-SCB module is nested under {\tt spacepy.pybats.ram}.
+% QUESTION: Steve, do we need to update the spacepy location (moved to git didn't it?)? Also, should we expand on this section to show actual examples of the visualization?
 
-Stand-alone code snippets for both Python and Matlab can be found in the
-{viz} directories.  These routines are largely out-of-date and not maintained.
-We recommend caution when using them.
+Stand-alone code snippets for both Python and Matlab can be found in the {viz} directories.  These routines are largely out-of-date and not maintained. We recommend caution when using them.
+% QUESTION: ^ This seems like a silly comment, maybe we should add new stand-alone code snippets that actually work, or possibly code snippets that use spacepy as an example? I have both.
```

Comment:
The `viz` directory that's referring to was removed. 

The viz directory that exists now is `Scripts/viz` and that has tools for visualizing the domain (from restart files) using VTK and Paraview.